docs/get_started/quickstart.mdx

```diff
@@ -422,39 +817,64 @@ This guide takes you through your first steps with Restate.
+                            ```
+                        </TabItem>
+                        <TabItem value={"quarkus"}>
+                            You are all set to start developing your service.
+                            Open the project in an IDE, run your service and let it listen on port `9080` for requests:
 
-            You are all set to start developing your service.
-            Open the project in an IDE and configure it to build with Gradle.
-            Run your service and let it listen on port `9080` for requests:
+                            ```shell
+                            quarkus dev
+                            ```
+                        </TabItem>
+                        <TabItem value={"vanilla"}>
+                            You are all set to start developing your service.
+                            Open the project in an IDE, run your service and let it listen on port `9080` for requests:
 
-            ```shell
-            ./gradlew run
-            ```
+                            ```shell
+                            mvn compile exec:java
+                            ```
+                        </TabItem>
+                    </Tabs>
+                </TabItem>
+                <TabItem value={"gradle"}>
+                    <Tabs groupId={"architecture"} className={"display-none"}>
+                        <TabItem value={"vanilla"} label={<><img src='/img/quickstart/java.svg' alt="Vanilla" className="icon" />Vanilla</>}>
+                            You are all set to start developing your service.
+                            Open the project in an IDE, run your service and let it listen on port `9080` for requests:
+
+                            ```shell
+                            ./gradlew run
+                            ```
+                        </TabItem>
+                    </Tabs>
+                </TabItem>
+            </Tabs>
 
         </Step>
 
         <Step stepLabel="4" title="Register the service">
 
             Tell Restate where the service is running, so Restate can discover and register the services and handlers behind this endpoint:
 
-            <CodeWithTabs>
+            <TerminalWithTabs>
 
-                ```shell !!tabs CLI
-                restate deployments register http://localhost:9080
-                ```
+                # !!terminals
 
-                ```shell !!tabs curl
-                curl localhost:9070/deployments -H 'content-type: application/json' \
-                -d '{"uri": "http://localhost:9080"}'
+                ```shell !command CLI
+                restate deployments register http://localhost:9080
                 ```
 
-            </CodeWithTabs>
-
-            <details className={"grey-details"}>
-
-                <summary>Output</summary>
-
-                ```shell
+                ```shell !output
                 ❯ SERVICES THAT WILL BE ADDED:
                 - Greeter
                 Type: Service
```

Comment:
The output of registration looks much nicer with schemas now, perhaps you can show that :shrug:

Reply:
ah but i think it's not available in released versions.